ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -66,6 +66,36 @@ def test_tensorflow_Acosh(  # NOQA
     )
 
 
+# Angle
+@handle_frontend_test(
+    fn_tree="tensorflow.raw_ops.Angle",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("complex"),
+    ),
+    Tout=st.sampled_from([ivy.float32]),
```

Comment:
It would be best to use `helpers.get_dtypes("float", full=False)` as this would pick any supported float data type.